lib/WatcherProcess.php

```diff
@@ -393,6 +393,10 @@ private function generateWorkerCommand(Console $console): string {
             $parts[] = $ini;
         }
 
+        if ($console->isArgDefined("worker-args")) {
+            $parts = array_merge($parts, explode(' ', $console->getArg("worker-args")));
```

Comment:
I'm not sure whether a simple `explode(' ', ...)` is the best way to go here. Does CLImate support one argument to appear multiple times? I don't think so, but that would be much better.